src/containers/Bookings.js

```diff
@@ -1,19 +1,49 @@
 import React, { Component } from "react";
 import Search from "../components/Search.js";
+import Results from "../components/Results.js";
+
 // import Results from "../components/Results.js";
-// import * as FakeBookings from "../data/fakeBookings.json";
+import * as FakeBookings from "../data/fakeBookings.json";
+
+
 
 export default class Bookings extends Component {
-  search = () => {
-    console.info("to do!");
-  };
+  constructor(props) {
+    super(props);
+    this.state = { searched: false };
+    this.bookingData = null;
+  }
+
   
+  handleSearchButtonClicks = (e, filter) => {
+    if (filter.customerId) {
+      this.bookingData = FakeBookings.filter(booking => booking.customerId = filter.customerId);
+    } else if (filter.customerName) {
+      this.bookingData = FakeBookings.filter(booking => booking.firstName.includes(filter.customerName));
+    } else {
+      this.bookingData = [...FakeBookings];
+    }
+    this.addNumberOfDays();
+    this.setState({searched: true});
+  };
+
+  addNumberOfDays = () => {
```

Comment:
Good job in breaking down the functions, makes it easier to read and write tests for